City-of-Helsinki/drupal-helfi-rekry
# UHF-5294: City description title and text

[UHF-5294](https://helsinkisolutionoffice.atlassian.net/browse/UHF-5294): Rekry: Ilmoituksen kaupunki-osion teksti

## What was done
- added fields in the settings form for city description title and text
- added an admin menu link for the settings form under tools
- added the city description title and text on a job listing page

## How to install

1. set up rekry instance
2. checkout this branch: `git checkout UHF-5294_city_description`
3. clear cache: `make drush-cr`

## How to test

- as an admin user, make sure you see the "Rekry content settings" menu item under "Tools → Site settings"
- from that page, access the translation interface with the "Translate rekry content settings" link
- in the translation interface, edit the "City description title" and "City decription text" values in different languages
- open any job listing and see the texts in the language of the content on the right
